services/course-material/src/components/chatbot/CitationPopover.tsx

```
@@ -0,0 +1,218 @@
+        setHoverPopperElement(true)
+      }}
+      onMouseLeave={() => {
+        setHoverPopperElement(false)
+      }}
+      onBlur={() => {
+        //unfocusPopperElement()
+      }}
+      {...popperAttributes.popper}
+    >
+      <div className={speechBalloonStyle}>
+        <p id={popDescribeId} dangerouslySetInnerHTML={{ __html: content }}></p>
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)